src/Paprika.Tests/Store/AbandonedTests.cs

```diff
@@ -250,4 +243,61 @@ public async Task Reuse_in_grow_and_shrink()
             await block.Commit(CommitOptions.FlushDataAndRoot);
         }
     }
+
+    [Test]
+    [Category(Categories.LongRunning)]
+    public async Task Abandoned_chain_creation_with_overflow()
+    {
+        // Minimum iterations required to overflow the abandoned list (AbandonedList.MaxCount).
+        const int minIterations = 60;
```

Comment:
This const is probably dependend on the `AbandonedList` max count, is it not? If it is, is should be taken from the `AbandonedList`. Otherwise, if we optimize the list and increase the bucket count it will provide false positive.